lib/networks.ts

```diff
@@ -21,7 +17,63 @@ import {
     Xai = 660279,
   }
   
-  export const xai: Chain = {
+
+
+
+  export interface TokenBridge {
```

Comment:
Is there a reason we have to declare all of this stuff vs importing it?

Reply:
Yeah, you shouldn't really need any of these except `L1Network` and `L2Network`, which are exported from the SDK. As long as you declare your object to be one of those types, the type system will guide you to fill out all the properties.